### PR TITLE
Tool to determine if a .nupkg has valid Microsoft metadata

### PR DESCRIPTION
**Pull request by joelverhagen**
This is a new version of https://github.com/NuGet/NuGetGallery/pull/7000. I did this so that I can leave `jver-verify` as-is, which is hard-coded into Arcade:
https://github.com/dotnet/arcade/blob/a68b528ad1d0305f55c4fbdc977ebb0c02cfcc19/eng/common/post-build/nuget-validation.ps1#L12

Given that Arcade is using this in their official pipeline, it's time to merge this and get them to switch to a build that is properly signed and published by the NuGet team. The current dependency is precarious since the binary they are using is built by me and not published by the NuGet team.
https://www.nuget.org/packages/Jver.VerifyMicrosoftPackage/0.0.1-beta

We will be providing the tool as-is (no behavior changes).

### CODE REVIEW

File: src/VerifyMicrosoftPackage/verify.ps1 [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+   [string]$DownloadPath,
+   [Parameter(ValueFromRemainingArguments = $true)]
+   [string[]]$args
+)
+
+# The URL to download nuget.exe.
+$nugetExeUrl = "https://dist.nuget.org/win-x86-commandline/v4.9.4/nuget.exe"
+
+# The package ID of the verification tool.
+$packageId = "Jver.VerifyMicrosoftPackage"
```
Comment (loic-sharma) [inline src/VerifyMicrosoftPackage/verify.ps1]: Is this the right package ID? Or should this be `NuGet.VerifyMicrosoftPackage`?

File: src/VerifyMicrosoftPackage/verify.ps1 [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+.PARAMETER args
+    Arguments that will be passed to the verification tool.
+.EXAMPLE
+    PS> .\verify.ps1 *.nupkg
+    Verifies the metadata of all .nupkg files in the currect working directory.
+.EXAMPLE
+    PS> .\verify.ps1 --help
+    Displays the help text of the downloaded verifiction tool.
+.LINK
+    https://github.com/NuGet/NuGetGallery/blob/jver-verify/src/VerifyMicrosoftPackage/README.md
```
Comment (loic-sharma) [inline src/VerifyMicrosoftPackage/verify.ps1]: Should this point to the `master` branch to align with the readme?

File: src/VerifyMicrosoftPackage/verify.ps1 (hunk 1 above)
Comment (joelverhagen) [reply src/VerifyMicrosoftPackage/verify.ps1]: You're right, `NuGet.VerifyMicrosoftPackage` is the correct one.

File: src/VerifyMicrosoftPackage/verify.ps1 (hunk 2 above)
Comment (joelverhagen) [reply src/VerifyMicrosoftPackage/verify.ps1]: Yes.

**Pull request merged**